### PR TITLE
Create diary card

### PR DESCRIPTION
**Pull request by udyogi-siphara**
[ITP-1760]

[ITP-1760]: https://fcodelabs.atlassian.net/browse/ITP-1760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/DiaryCard.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import {Button,Typography,CardContent,CardActions, Card} from '@mui/material';
+import  React  from "react";
+import { useState } from 'react';
+
+const DiaryCard = (props:any) => {
```
Comment (udaraJ08) [inline src/components/DiaryCard.tsx]: Do not use any. Define a data type by an interface

File: src/components/DiaryCard.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    //use state for show more btn
+    const [showMore, setShowMore] = useState(false);
+
+    // for setthe show more,hide
+    const handleToggleShowMore = () => {
+        setShowMore(!showMore);
+    };
+
+    // find the characters of description less than 100
+    const renderDescription = () => {
```
Comment (udaraJ08) [inline src/components/DiaryCard.tsx:24]: Move this function to a utility file.

**Review by udaraJ08 — CHANGES_REQUESTED** [review]

File: src/components/DiaryCard.tsx (hunk 1 above)
Comment (udyogi-siphara) [reply src/components/DiaryCard.tsx]: updated

File: src/components/DiaryCard.tsx (hunk 2 above)
Comment (udyogi-siphara) [reply src/components/DiaryCard.tsx:24]: updated

**Review by shamal-iroshan — APPROVED** [review]

**Pull request merged**